app/controllers/api/v1/replacement_document_validation_requests_controller.rb

```diff
@@ -47,6 +47,18 @@ def update
 
       private
 
+      def check_file_type
+        unless Document::PERMITTED_CONTENT_TYPES.include? params[:new_file].content_type
+          render json: { message: "The file type must be JPEG, PNG or PDF" }, status: :bad_request
```

Comment:
It will be great if we can also update the AWS policy to allow only these certain file types to be uploaded to my Amazon S3 bucket using our terraform